pkg/cmd/roachtest/scrub.go

```diff
@@ -57,9 +57,9 @@ func makeScrubTPCCTest(
 	for i := 0; i < len(stmts); i += len(tpccTables) {
 		for j, t := range tpccTables {
 			if indexOnly {
-				stmts[i+j] = fmt.Sprintf(`EXPERIMENTAL SCRUB TABLE tpcc.%s WITH OPTIONS INDEX ALL`, t)
+				stmts[i+j] = fmt.Sprintf(`EXPERIMENTAL SCRUB TABLE tpcc.%s AS OF SYSTEM TIME '-5s' WITH OPTIONS INDEX ALL`, t)
```

Comment:
Have you tried running it with '-1ms' to see if providing it with a specific timestamp itself is enough as opposed to running it in the past.